plugins/check_apt.c

```diff
@@ -148,22 +153,23 @@ int process_arguments (int argc, char **argv) {
 		{"include", required_argument, 0, 'i'},
 		{"exclude", required_argument, 0, 'e'},
 		{"critical", required_argument, 0, 'c'},
+		{"list", no_argument, 0, 'l'},
 		{"input-file", required_argument, 0, INPUT_FILE_OPT},
 		{0, 0, 0, 0}
 	};
 
 	while(1) {
-		c = getopt_long(argc, argv, "hVvt:u::U::d::ni:e:c:", longopts, NULL);
+		c = getopt_long(argc, argv, "hVvt:u::U::d::ni:e:c:l", longopts, NULL);
 
 		if(c == -1 || c == EOF || c == 1) break;
 
 		switch(c) {
 		case 'h':
 			print_help();
-			exit(STATE_UNKNOWN);
+			exit(STATE_OK);
```

Comment:
Please never do that, this might lead to plugins which only print the help or version and you will never notice because they exit OK. See https://www.monitoring-plugins.org/doc/guidelines.html#AEN78